### ISSUE TITLE
Augs in validation set

### ISSUE THREAD

**Issue by ariG23498** [soumik12345/mirnetv2#19]
https://github.com/soumik12345/mirnetv2/blob/0dd215b926f772f34c7ed208f0ac7fc24e7e4af4/mirnetv2/dataloader/base/base_dataloader.py#L118-L119

IMO we need to augment the val dataset. We also need another set for test, which will have no augs.

**Comment by soumik12345** [comment]
@ariG23498  Won't the application of random augmentations on the validation dataset mean that the validation data would change every epoch? Would that give us a fair estimate of the model's performance during training?

**Comment by ariG23498** [comment]
Do you mean that validation data should be as close to the testing data as possible? I try with overlapping the data distribution of train and val, so that I understand how my algo (loss and optimizers) perform.

But with the test and val data distribution being similar, you are trying to assess the algo on the task.

We can keep the val without augs, but we would _absolutely_ need a test set here.

**Comment by soumik12345** [comment]
> Do you mean that validation data should be as close to the testing data as possible? I try with overlapping the data distribution of train and val, so that I understand how my algo (loss and optimizers) perform.
> 
> But with the test and val data distribution being similar, you are trying to assess the algo on the task.
> 
> We can keep the val without augs, but we would _absolutely_ need a test set here.

@ariG23498 We can use the [eval15 split](https://wandb.ai/ml-colabs/mirnet-v2/artifacts/dataset/lol-dataset/v0/files/eval15) of the LoL dataset as our test set.
